test/automation/src/sql/configurePythonDialog.ts

```diff
@@ -33,7 +33,13 @@ export class ConfigurePythonDialog extends Dialog {
 		await this.code.waitForElement(installButton);
 		await this.code.dispatchKeybinding('enter');
 
-		return this.waitForDialogGone();
+		await this.waitForDialogGone();
+		return this._waitForInstallationComplete();
+	}
+
+	private async _waitForInstallationComplete(): Promise<void> {
+		const installationCompleteNotification = '.notifications-toasts div[aria-label="Notebook dependencies installation is complete, source: Notebook Core Extensions (Extension), notification"]';
```

Comment:
Nothing else we can use except the aria label? We really shouldn't be relying on that for tests.

Reply:
The notification toasts have an id but that id changes based on how many notifications are present.

Reply:
Can you use substring attribute selectors for that? https://stackoverflow.com/questions/13533484/css-select-elements-with-partial-id

Reply:
All the ids start with "list_id_" so I wouldn't be able to select the specific notification using substring attribute selectors.